libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -71,11 +96,32 @@ describe('PageComponent', () => {
   });
 
   it('should render back button with correct size', async () => {
+    spectator.detectChanges();
     await TestHelper.whenReady(ionToolbar);
     const ionBackButton = spectator.queryHost('ion-toolbar ion-buttons ion-back-button');
     expect(ionBackButton).toHaveComputedStyle({
       width: fatFingerSize,
       height: fatFingerSize,
     });
   });
+
+  it('should hide tab bar when hideTabs is true on enter', () => {
+    spectator.component.hideTabs = true;
+    spectator.detectChanges();
+    eventSubject.next(new NavigationEnd(1, '123', '123'));
+
+    expect(tabbar.hide).toHaveBeenCalled();
+  });
+  it('should show tab bar when hideTabs is true on leave', () => {
+    spectator.component.hideTabs = true;
+    spectator.detectChanges();
```

Comment:
Do you need to call change detection?
If you do need it, consider using `spectator.setInput` instead:
```suggestion
    spectator.setInput('hideTabs', true);
```